mkdocs.yml

```
@@ -195,6 +195,7 @@ plugins:
         builders/build/substrate-api/py-substrate-interface.md: builders/substrate/libraries/py-substrate-interface.md
         builders/build/substrate-api/sidecar.md: builders/substrate/libraries/sidecar.md
         builders/build/substrate-api/overview.md: builders/substrate/overview.md
+        learn/dapps-list/state-of-the-dapps.md: learn/dapps-list/dapp-directory.md
```

```suggestion
        learn/dapps-list/state-of-the-dapps.md: learn/dapps-list/index.md
```

Let's redirect to the index page, just in case there are any backlinks and that way it's clear that State of the DApps isn't available anymore